src/main/webapp/components/BaseFormView.jsx

```diff
@@ -537,7 +537,7 @@ class BaseFormView extends PureComponent {
     saveData = () => {
         const body = new URLSearchParams();
         Object.keys(this.datadict).forEach((key) => {
-            if (this.datadict[key] != null) {
+            if (this.datadict[key] != null && key != 'redirect_url') {
```

Comment:
Correct me but I think it's wrong logically. `||` condition would be correct as we also need to present passing `null` value fields.